bsip-0048.md

```diff
@@ -10,45 +10,61 @@
 
 # Abstract
 
-The possibility of creating assets on the BitShares Blockchain is a core feature
-of our platform. In the case of an ico, the ico-holders can create an asset with
-a fixed maximum supply. However this maximum supply can be changed by the
-asset creator (issuer) anytime. This exposes participators ico to the risk,
-of loosing value of their bought asset.
+The possibility of creating assets on the BitShares Blockchain is a core feature of our platform. However some things are too maleable when they should not be. Other things are inflexible when they should be.
```

Comment:
Typo: mal*l*eable

Reply:
> Other things are inflexible when they should be.

This sounds wrong. Perhaps better "are not flexible when they should be".

Reply:
Fixed